lib/rubocop/cop/style/if_with_boolean_literal_branches.rb

```diff
@@ -76,6 +90,12 @@ def on_if(node)
 
         private
 
+        def multiple_elsif?(node)
```

Comment:
We might add this to `rubocop-ast` as well.

Reply:
That's a good idea! I will open a PR to rubocop-ast.

Reply:
I've opened https://github.com/rubocop/rubocop-ast/pull/239 to rubocop-ast.

Reply:
The above PR has been closed because there are currently no plans to use it outside of this PR. I would like to extract to rubocop-ast when a concrete reuse use case comes up.